### DISCUSSION TITLE
Disable prev/next when slides only contain one image?

### DISCUSSION THREAD

**Discussion by ivanyufen** [igordanchenko/yet-another-react-lightbox#250] [Q&A] [answered]
i found that the prev/next keep repeating the same image since my slides only contain one image. Any way to disable this? thank you

**Comment by igordanchenko** [comment, marked as answer]
Certainly. You just need to disable "infinite" carousel mode in this case:

```jsx
<Lightbox
  carousel={{ finite: slides.length <= 1 }}
  // ...
/>
```

You can also hide navigation buttons when there is just one slide:

```jsx
<Lightbox
  carousel={{ finite: slides.length <= 1 }}
  render={{
    buttonPrev: slides.length <= 1 ? () => null : undefined,
    buttonNext: slides.length <= 1 ? () => null : undefined,
  }}
  // ...
/>
```

This use case is covered in the documentation over here - https://yet-another-react-lightbox.com/customization#HidingNavigationButtons
